packages/datadog-instrumentations/src/mocha/utils.js

```diff
@@ -0,0 +1,305 @@
+const {
```

Comment:
ℹ️ this file now contains functions used both for parallel and serial mode and for the main and worker processes